debian/assets/carlos_ctl/tests/test_etl_sql.py

```
@@ -0,0 +1,606 @@
+    for ch in body:
+        if ch == "(":
+            depth += 1
+        elif ch == ")":
+            depth -= 1
+        if ch == "," and depth == 0:
+            exprs.append(cur.strip())
+            cur = ""
+        else:
+            cur += ch
+    exprs.append(cur.strip())
+    return dict(zip(targets, exprs))[target_col]
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)